test/e2e/artifact_model_1.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "MNIST",
+    "uri": "gs://my-bucket/mnist",
+    "workspace": {
```

Comment:
I think this will not work. [Service.go](https://github.com/kubeflow/metadata/blob/aec51c4d3e4803618a3624e38273a1ad8f16424b/service/service.go#L142) implements the workspace as port of the name.  
Example: name = `workspace1/alpha/MNIST` -> workspace = `workspace1/alpha` and name = `MNIST`